Kconfig.nrf

```diff
@@ -16,14 +33,19 @@ config BOARD_THINGY91_NRF9160NS
 #  - For CC3XX RNG
 config MAIN_STACK_SIZE
 	default 2048 if ZTEST
-	default 2048 if ENTROPY_CC3XX
+	default 2048 if ENTROPY_CC3XX && !(SPM || BUILD_WITH_TFM)
 
 config ZTEST_STACKSIZE
 	default 2048 if ZTEST
 
 config PRIVILEGED_STACK_SIZE
 	default 2048 if ZTEST
 
+# Set ENTROPY_GENERATOR to true for TF-M or SPM builds with enabled
+# RNG provided from secure services.
+config ENTROPY_GENERATOR
+	default y if BUILD_WITH_TFM || (SPM && SPM_SECURE_SERVICES)
```

Comment:
Is there a config in TFM as well that can trun RNG on or off?

Reply:
We have not provided one and we argue that providing it will just cause security issues because then you don't have a proper RNG in the system